notebooks/Tutorial_templates/tutorial_template_msb_v1.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
<p>Is the image (Fig. 1) just an example of what you could add under section 1? Given that we also show the logo above, this image could be confused as something that should always be required here, without some additional explanation. For example, maybe we're trying to demonstrate the best practice for how to include images under a separate /img file? If so, we should add that additional guidance. It may also help to indicate which of these sections should generally always be included versus those that are more optional. For example, maybe the example of an end product may not be relevant in all cases.</p><p> </p><p>Under 1.1:</p><ul><li>I like including a reference to the environment. Note that in all of our tutorial folders we have a ReadMe that gives an overview of the notebook including Setup instructions, and this points back to the Readme of the main repo. I think it's helpful to include a note on the environment here but we should always point back to the Readme (https://github.com/nsidc/NSIDC-Data-Tutorials#readme) so that we're not repeating info in different places (otherwise our guidance will not stay in sync if we need to update it). See AC in CRYO-52: "Prereq instructions for creating the tutorial environment are included and point to the main repo readme for setup instructions"</li><li>So, since I also recommend we broaden section 2 as a Prerequisite section, let's combine information on environment setup and Earthdata login under a single section. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>